check_openapi_generator_has_been_run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+# First attempt to generate new code
+make generate code/fix
+
+# Then make sure that no new code was generated 
+changed_file_count=$(git status -s | wc -l)
```

Comment:
I am being very nitpicky already but a possible alternative is using `git status --porcelain`.
The description of the `porcelain` flag is:
```
       --porcelain[=<version>]
           Give the output in an easy-to-parse format for scripts. This is similar to the short output, but will remain stable across Git versions and regardless of user configuration. See below for details.

           The version parameter is used to specify the format version. This is optional and defaults to the original version v1 format.
```

Up to you how to implement it. Both approaches are good

Reply:
Good to know, thanks.